server/xml.server.php

```diff
@@ -34,7 +34,7 @@
 
 /* Set the correct headers */
 header("Content-type: text/xml; charset=" . AmpConfig::get('site_charset'));
-header("Content-Disposition: attachment; filename=information.xml");
+// header("Content-Disposition: attachment; filename=information.xml");
```

Comment:
IMO the header all together should be removed, but the header is commented because it makes it much easier to debug.